job_templates/package_action_-_ssh_default.erb

```diff
@@ -69,6 +69,12 @@ handle_zypp_res_codes () {
 
   if [ "${ZYPP_RES_CODES[$RETVAL]}" != "" ]; then
     echo ${ZYPP_RES_CODES[$RETVAL]}
+
+    # Set flag for later usage in katello tracer upload
+    if [ "$RETVAL" == "102" ]; then
+        touch /var/run/reboot-needed
```

Comment:
Since Debian uses `reboot-required` and I left a comment in the other PR I added the After Core Merge label.